src/main/java/machines/Excavator.java

```diff
@@ -0,0 +1,11 @@
+package machines;
+
+public class Excavator extends Machine {
+    public void doWork() {
+        System.out.println("Excavator has started the job");
+    }
+
+    public void stopWork() {
+        System.out.println("Excavator has stopped the job");
```

Comment:
Similarly, the message 'Excavator has stopped the job' should be more informative. You might want to use 'Excavator has finished working' to make it clear that the Excavator has stopped.